docs/start/routing.md

```diff
@@ -31,36 +31,36 @@ export default defineRoutes([
     route(":city", "./concerts/city.tsx"),
     route("trending", "./concerts/trending.tsx"),
   ]),
-]);
+];
 ```
 
 ## File System Routes
 
 If you prefer a file system routing convention, you can use the convention provided with Remix v2, but you can also make your own.
 
 ```tsx filename=app/routes.ts
-import { defineRoutes } from "@react-router/dev/routes";
+import { type RoutesConfig } from "@react-router/dev/routes";
 import { remixRoutes } from "@react-router/remix-v2-routes";
 
-export default defineRoutes(await remixRoutes());
+export const routes: RoutesConfig = remixRoutes();
```

Comment:
Now that the export type is a little clearer, I'm removing the redundant `await` on the export since we handle promises within the Vite plugin.